models/product.model.lkml

```diff
@@ -215,3 +215,20 @@ explore: rpt_active_user_base {
   label: "Active User Base"
   group_label: "[New] Active Users"
 }
+
+explore: fct_subscription_history {
+  label: "Cloud Subscription History (360)"
+  group_label: "[New] Cloud 360"
+
+  join: dim_self_serve_customers {
+    relationship: many_to_one
+    type:  left_outer
+    sql_on:  ${fct_subscription_history.customer_id} = ${dim_self_serve_customers.customer_id} ;;
+  }
+
+  join: dim_server_info {
+    relationship: many_to_many
+    type: full_outer
```

Comment:
Why full outer join? Isn't a left outer join preferred? `dim_server_info` might contain more records than `fct_subscription_history`.

2/5 `dim_server_info` primary key is `server_id`. Not sure how such joins might behave.